src/intermediate/Branching.cpp

```diff
@@ -48,6 +48,11 @@ const Local* BranchLabel::getLabel() const
     return assertArgument(0).local;
 }
 
+Local* BranchLabel::getLabel()
+{
+    return getArgument(0)->local;
```

Comment:
use assertArgument(0) here, which saves us from creating an Optional object and one copy of Value